### ISSUE TITLE
Can Not Create Generate Draft Model Node

### ISSUE THREAD

**Issue by Uniky** [VAST-AI-Research/ComfyUI-Tripo#23]
I have completely reinstalled ComfyUI, with only this custom node and the manager installed, but still, I get an undefined node, which appears to be the Generate Draft Model node, because I can create other nodes in this package.

![p1](https://github.com/user-attachments/assets/025f7fd5-e61c-4e57-add9-dba31a293f84)
![p2](https://github.com/user-attachments/assets/92da3d10-0831-421c-8160-0ef18cef29df)
![p3](https://github.com/user-attachments/assets/9998b461-1333-4f01-91f0-a31ae8d97b6f)
![p4](https://github.com/user-attachments/assets/954f9231-4f61-4740-9d7e-a845a049d0b1)

**Comment by seed93** [comment]
OK. Let me update the workflow image. Before that, I think you can get the node from menu.

**Comment by Uniky** [comment]
> OK. Let me update the workflow image. Before that, I think you can get the node from menu.

Not really, I can't create the Generate Draft Model node even from the menu, nothing appears. As I mentioned before, I can only create the other five nodes from this package.

**Comment by Uniky** [comment]
It seems that ComfyUI disqualifies the format ("INT",)
